### DISCUSSION TITLE
Cannot type capital letters or special characters on Rustdesk remote; both are Windows machines

### DISCUSSION THREAD

**Discussion by oldmike60** [rustdesk/rustdesk#6568] [General]
Host:  WIndows 11
Remote:  Windows 10

Typing a capital letter on Remote does nothing; no error, cursor does not advance, it's like I did not type anything
Upper case is ignored using SHIFT key and with CAPT LOCK.  Same for special characters

**Comment by oldmike60** [comment]
BTW:  if I type what I want (password that includes mixed case and some special characters) into Notebook, copy, switch to Remote, then pasted the PW in, it works fine.  So seems to be strictly limited to the keyboard.

**Comment by thestormdragon** [comment]
Same issue issue but both devices are Windows 11

**Comment by Crypto69** [comment]
I have the same issue
